packages/main/src/Dialog.js

```diff
@@ -201,21 +214,32 @@ class Dialog extends Popup {
 		this._isRTL = this.effectiveDir === "rtl";
 		this.onPhone = isPhone();
 		this.onDesktop = isDesktop();
+
+		ResizeHandler.deregister(this, this._screenResizeHandler);
```

Comment:
Register/deregister these events in onEnterDOM/onExitDOM methods

	onEnterDOM() {
		ResizeHandler.register(this, this._screenResizeHandler);
		ResizeHandler.register(document.body, this._screenResizeHandler);
	}

	onExitDOM() {
		ResizeHandler.deregister(this, this._screenResizeHandler);
		ResizeHandler.deregister(document.body, this._screenResizeHandler);
	}

Reply:
Done